third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -2480,18 +2480,6 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
         ) = &args[1].value
         {
             let field_name = self.symbol_pool().make(name.value.as_str());
-            let constraint =
```

Comment:
You can't just remove this, because there will be no error reported anylonger if the field doesn't exist. See line 2490 assuming the error that the field does not exist would have been already reported.

Reply:
done